Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:2.6
```

Comment:
No reason you can't use the Debian base image, but I'm running the alpine image for GeoWeb in production and it seems to do fine.